packages/opentelemetry-core/README.md

```diff
@@ -1,3 +1,16 @@
 # OpenTelemetry Core
 
-TODO
+This package provides concrete implementations of the OpenTelemetry types for trace and metrics. It's intended for use both on the server and in the browser.
+
+## Useful links
+- For more information on OpenTelemetry, visit: <https://opentelemetry.io/>
+- For more about OpenTelemetry Node: <https://github.com/open-telemetry/opentelemetry-js>
```

Comment:
Minor: OpenTelemetry Node to OpenTelemetry JavaScript (or Node/JavaScript) ?

Reply:
I think `OpenTelemetry JavaScript` is good to me